### DISCUSSION TITLE
Welcome to indi_pylibcamera Discussions!

### DISCUSSION THREAD

**Discussion by scriptorron** [scriptorron/indi_pylibcamera#3] [Announcements]
# Welcome!
You are welcome to use Discussions for:
* giving me feedback or telling the experiences you made
* discussing features (missing or already implemented)
* and all the other topics which are not issues.

**Comment by Vados1er** [comment]
Hello
Did indi_pylibcamera works with StellarMate ?

**Reply by scriptorron** [reply]
I do not have Stelarmate and can not test the driver with it. The version of the `libcamera` and `picamera2` packages are critical for the driver. About 6 months ago the packages in the Stelarmate repository were too old. I am not sure if they have updates now. See issue #1.
